mungegithub/github/github.go

```diff
@@ -1216,6 +1218,27 @@ func (obj *MungeObject) AssignPR(owner string) error {
 	return nil
 }
 
+// CloseIssuef will close the given issue with a message
+func (obj *MungeObject) CloseIssuef(format string, args ...interface{}) error {
+	config := obj.config
+	msg := fmt.Sprintf(format, args...)
+	if err := obj.WriteComment(msg); err != nil {
+		return fmt.Errorf("failed to write comment to %v: %q: %v", *obj.Issue.Number, msg, err)
```

Comment:
It looked like closing PRs and issues were different :(

On Thu, May 26, 2016 at 11:24 AM, Eric Paris <notifications@github.com>
wrote:

> In mungegithub/github/github.go
> <https://github.com/kubernetes/contrib/pull/1061#discussion_r64795700>:
>
> > @@ -1216,6 +1218,27 @@ func (obj *MungeObject) AssignPR(owner string) error {
> >  	return nil
> >  }
> >
> > +// CloseIssuef will close the given issue with a message
> > +func (obj *MungeObject) CloseIssuef(format string, args ...interface{}) error {
> > +	config := obj.config
> > +	msg := fmt.Sprintf(format, args...)
> > +	if err := obj.WriteComment(msg); err != nil {
> > +		return fmt.Errorf("failed to write comment to %v: %q: %v", *obj.Issue.Number, msg, err)
>
> also, do you know if ClosePR() could be abandoned and use this function
> instead? I'll try to remember to look sometime.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1061/files/e481c7373b54ad4eee2214188a0f5adfa809b3fb#r64795700>
>